application/electron/src/services/files/service.file.picker.ts

```diff
@@ -0,0 +1,128 @@
+import ServiceElectron, { IPCMessages } from '../service.electron';
+import ServiceStreams, { IStreamInfo } from '../service.streams';
+import ServiceStorage, { IStorageScheme } from '../service.storage';
+import ServiceStreamSource from '../service.stream.sources';
+import ServiceHotkeys from '../service.hotkeys';
+import { getDefaultFileParser, AFileParser, getParserForFile } from '../../controllers/files.parsers/index';
+import { IMapItem, ITicks } from '../../controllers/files.parsers/interface';
+import { dialog, OpenDialogReturnValue, FileFilter } from 'electron';
+import Logger from '../../tools/env.logger';
+import * as Tools from '../../tools/index';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Subscription } from '../../tools/index';
+import { IService } from '../../interfaces/interface.service';
+
+/**
+ * @class ServiceFilePicker
+ * @description Opens files and returns fullpath
+ */
+
+class ServiceFilePicker implements IService {
+
+    private _logger: Logger = new Logger('ServiceFilePicker');
+    // Should detect by executable file
+    private _subscription: { [key: string]: Subscription } = {};
+    private _options: any;
```

Comment:
why do you need this field? seems not to be used

Reply:
you are right. It isn't used. Will be removed